app/views/tag/_miniCard.html.erb

```diff
@@ -26,7 +30,7 @@
 
     <% if current_user && current_user.following(tag.name) %>
       <a class="pull-right btn btn-primary requireLogin" href="/post?tags=<%= tag.name %>">Post <i class="fa fa-plus fa-white"></i></a>
-    <% else %>
+    <% elsif !tag.name.include? ':' %>
```

Comment:
I saw a method you have used in another branch `power_tag`. Is it available here?

Reply:
It should be; it should be shown in /app/models/tag.rb

Reply:
I'm not sure why but that isn't working. It says undefined method `power_tag`